bundles/org.openhab.binding.sonos/src/main/resources/OH-INF/i18n/sonos.properties

```diff
@@ -199,6 +201,10 @@ channel-type.sonos.zonegroupid.description = Id of the Zone Group the Sonos devi
 channel-type.sonos.zonename.label = Zone Name
 channel-type.sonos.zonename.description = Name of the Zone Group associated to the Sonos device
 
+# channel types
+
+channel-type.sonos.codec.state.option.dolbyAtmos = Dolby Atmos
```

Comment:
To be removed.

Reply:
Well that's an odd thing for the script to move that down there when it changed it above.  Commit pushed to remove that offending line.